

const ComponentWithError = () => {
  throw new Error("This is a deliberate error for testing purpose.");
};

export default ComponentWithError;
